cars/v1/vanilla/templates/config/jvm.options

```diff
@@ -39,8 +39,6 @@
 8-13:-XX:CMSInitiatingOccupancyFraction=75
 8-13:-XX:+UseCMSInitiatingOccupancyOnly
 14-:-XX:+UseG1GC
-14-:-XX:G1ReservePercent=25
-14-:-XX:InitiatingHeapOccupancyPercent=30
```

Comment:
These options are still defined in the case where we enable G1 explicitly (see a few lines below) and I think they need to be removed there as well?

Reply:
good catch, thank you! I will push an update